requirements.txt

```diff
@@ -11,12 +11,16 @@ Pillow                  ==8.0.1       # HPND
 ffmpeg-python           ==0.2.0       # GNU-GPL
 simpleaudio             ==1.0.4       # MIT
 
-# Dependencies needed for converting Pytorch models to CoreML
-tensorflow              ==1.14.0
-keras                   ==2.2.4
-coremltools             ==3.3
-h5py                    ==2.10.0
+# Dependencies needed for converting Pytorch models to tensorflow Lite
+tensorflow              ==2.3.1
+Keras                   ==2.4.3
```

Comment:
I know the file wasn't complete with licenses before, but can we agree to add license information here from now on? Both tensorflow and Keras are: `Apache 2.0`